src/main/kotlin/pl/edu/agh/gem/external/dto/InternalExchangeRateResponse.kt

```diff
@@ -0,0 +1,19 @@
+package pl.edu.agh.gem.external.dto
+
+import pl.edu.agh.gem.internal.model.ExchangeRate
+import java.math.BigDecimal
+import java.time.Instant
+
+data class InternalExchangeRateResponse(
```

Comment:
do we want to return currencyFrom, currencyTo, createdAd?  In expense manager i only need value of rate field